README.md

```diff
@@ -1,7 +1,54 @@
 [comment]: # ( Copyright Contributors to the Open Cluster Management project )
 # Get started 
 
+## Config file
+By default, the multicluster controlplane would try to find and open the config file named `ocmconfig.yaml` in the current directory. And the config file can be indicated by setting the flag `--config-file=<configfile.yaml>` while starting the multicluster controlplane.
```

Comment:
> By default, the multicluster controlplane would try to find and open the config file named `ocmconfig.yaml` in the current directory. And the config file can be indicated by setting the flag `--config-file=<configfile.yaml>` while starting the multicluster controlplane.

By default, the multicluster controlplane would try to find `ocmconfig.yaml` in the current directory. You can use the customized one by setting the flag `--config-file=<config-file.yaml>`. 

Here is a simple file of `config-file.yaml`:

Reply:
done